gramex/apps/nlg/gramex.yaml

```diff
@@ -7,6 +15,16 @@ import:
     YAMLURL: $YAMLURL/languagetool/
 
 url:
+  nlg-home-$*:
+    pattern: /$YAMLURL/
+    handler: FileHandler
+    kwargs:
+      path: $YAMLPATH/index.html
+      transform:
```

Comment:
Possible alternative: use `template: true` and `{% set nlg_base = globals().get('nlg_base', '') %}` in the template